cfgov/ask_cfpb/migration_scripts/migrate_knowledgebase.py

```diff
@@ -0,0 +1,281 @@
+from __future__ import unicode_literals
+
+import time
+
+from knowledgebase.models import QuestionCategory as QC
+from knowledgebase.models import Question, Audience, UpsellItem, EnglishAnswer
+from ask_cfpb.models import (
+    Answer,
+    # AnswerPage,
+    Category,
+    NextStep,
+    SubCategory)
+from ask_cfpb.models import Audience as ASK_audience
+
+
+def get_kb_statuses(ask_id):
+    q = Question.objects.get(id=ask_id)
+    return {'en': (q.get_english_answer_state() == 'APPROVED'),
+            'es': (q.get_spanish_answer_state() == 'APPROVED')}
+
+
+def create_answer_pages(queryset):
+    counter = 0
+    if queryset:
+        for answer in queryset:
+            kb_statuses = get_kb_statuses(answer.id)
+            for lang in ['en', 'es']:
+                if kb_statuses[lang]:
+                    page = answer.create_or_update_page(language=lang)
+                    revision = page.get_latest_revision()
+                    revision.publish()
```

Comment:
revision.publish isn't our method, it's via wagtail. We might need figure out another way.